generator/script/CustomRenderer.js

```diff
@@ -50,7 +50,7 @@ exports.render_upgrade = function (color) {
 
         code = code.replace(/\n$/, '') + '\n';
 
-        return `<pre><code class="code" style="${color}">` + (escaped ? code:escape(code,true)) + `</code></pre>`;
+        return `<pre style="background-color: ${color[0]}"><code class="code">` + (escaped ? code:escape(code,true)) + `</code></pre>`;
```

Comment:
why to use concatenation in template string ? 

```suggestion
                return `<pre style="background-color: ${color[0]}"><code class="code">${escaped ? code:escape(code,true)}</code></pre>`;
```

apply this approach for all the file content